app/policies/group_policy.rb

```diff
@@ -15,6 +15,10 @@ def edit?
   alias_method :update?, :edit?
   alias_method :add_editor?, :edit?
 
+  def add_group_admin?
+    user.super_admin? || user.is_organisations_admin?(record.organisation)
+  end
```

Comment:
Is this one we can add as an alias, as `upgrade?` has the same permissions check?